packages/next/cli/next-start.ts

```diff
@@ -58,10 +62,20 @@ const nextStart: cliCommand = (argv) => {
     port = 0
   }
 
+  const keepAliveTimeout = args['--keepAliveTimeout']
+    ? parseInt(args['--keepAliveTimeout'])
+    : undefined
```

Comment:
This needs to have error handling when the user passes a non-integer

Also needs tests

Reply:
Should it throw or swallow the error? I figured an invalid value should throw so the user would know immediately. And what would throw there is a TypeError saying something about NaN. How do you think it should be handled?

Reply:
I'm checking for NaN for now and exiting if invalid.